lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -418,7 +412,7 @@ spec = do
             |] `shouldBe` (Left @String @(ApiT AddressPoolGap) msg)
 
         it "ApiT AddressPoolGap (not a integer)" $ do
-            let msg = "Error in $: expected Integer, encountered floating number\
+            let msg = "Error in $: parsing Integer failed, unexpected floating number\
```

Comment:
Probably few integration tests will have to be updated also...

Reply:
Yes, I am running them locally at the moment, waiting for them to complete before going through all the cases.

Reply:
```
1149 examples, 151 failures, 1 pending
```

Good.